### DISCUSSION TITLE
How to Run Multiple Instances on Host Mode

### DISCUSSION THREAD

**Discussion by charlestephen** [ZoeyVid/NPMplus#1130] [Q&A] [answered]
I used the IPV4 BIND environmental variables but still the IPV6 BIND variables impede me running multiple instances despite me disabling wherever I could IPV6. Do you have explicit instructions on how to do this?

**Comment by Zoey2936** [comment]
Can you show me both compose files?

**Reply by charlestephen** [reply, marked as answer]
Nvm, I found a workaround by running one container in host mode and another as a bridged container.

**Reply by Zoey2936** [reply]
ipv6 disabling is fixed now
